omisego-sdk/src/main/java/co/omisego/omisego/websocket/listener/SimpleSocketCustomEventListener.kt

```diff
@@ -16,17 +16,17 @@ import co.omisego.omisego.websocket.strategy.FilterStrategy
  * A [SocketCustomEventListener] that can be filtered for a specific type of event.
  */
 
-abstract class SimpleSocketCustomEventListener<Event : SocketEvent<*>> : SocketCustomEventListener {
+abstract class SimpleSocketCustomEventListener : SocketCustomEventListener {
     abstract val strategy: FilterStrategy
 
     @Suppress("UNCHECKED_CAST")
```

Comment:
You can now remove this suppress warning

Reply:
Yeah